src/base/Assert.cc

```diff
@@ -156,10 +156,11 @@ RuntimeError::RuntimeError(const std::string& msg) : std::runtime_error(msg) {}
     }
 
     msg << "celeritas: " << color_code('R') << "runtime error: ";
-    if (verbose_message)
+    if (verbose_message || detail.empty())
```

Comment:
Do you mean
```suggestion
    if (verbose_message || !detail.empty())
```
? (The current conditional expression seems to say that all message without detail will now be printed in verbose mode?)

Reply:
Right -- if you declare `CELER_VALIDATE(foo != bar,);` since there's no descriptive text it'll always print the failing code.